File: plugins/kotlin/j2k/shared/tests/testData/newJ2k/detectProperties/SynchronizedPropertyGetter.java
class C {
    private String name = "";

    public synchronized void start(String initialName) {
        name = initialName;
    }

    protected void touchName() {
        if (name != "") {
            System.out.println("Name isn't empty");
        }
    }

    public synchronized String getName() {
        return name;
    }
}

File: plugins/kotlin/j2k/shared/tests/testData/newJ2k/detectProperties/SynchronizedPropertyGetter.kt
internal class C {
    private var name = ""

    @Synchronized
    fun start(initialName: String) {
        name = initialName
    }

    protected fun touchName() {
        if (name !== "") {
            println("Name isn't empty")
        }
    }

    @Synchronized
    fun getName(): String {
        return name
    }
}
